src/execution_plan/ops/op_node_by_label_scan.h

```diff
@@ -22,6 +22,7 @@ typedef struct {
 	unsigned int nodeRecIdx;    /* Node position within record. */
 	GxB_MatrixTupleIter *iter;
 	Record child_record;        /* The Record this op acts on if it is not a tap. */
+	const char *label;          /* The label to look after */
```

Comment:
Why this change?

Reply:
`MATCH (a), (b) where (a)->[]->(b:B)` requires a label scan for b. since the query graph node in the sub-execution plan is the same node as in the main execution plan, it has no label, and modification of the main exection plan node is forbiden.